os/kernel/debug/sysdbg.c

```diff
@@ -208,20 +115,23 @@ static void simulate_undef_abort(void)
 
 static void show_usage(void)
 {
-	dbg("USAGE:\n");
-	dbg("sysdbg enable_monitor  /* Allocates memory and enables monitoring */\n");
-	dbg("sysdbg disable_monitor /* Frees the memory and disables monitoring */\n");
-	dbg("sysdbg read   /* Reads /dev/sysdbg to view debug information*/\n");
-	dbg("sysdbg max_task_count <count> /* Enter max_task_count */\n");
-	dbg("sysdbg max_irq_count <count> /* Enter max_irq_count */\n");
-	dbg("sysdbg max_sem_count <count> /* Enter max_sem_count */\n");
-	dbg("sysdbg data_abort /* Simulate data abort */\n");
-	dbg("sysdbg prefetch_abort /* Simulate Prefetch abort */\n");
-	dbg("sysdbg undef_abort  /* Simulate Undefined abort */\n");
-	dbg("sysdbg usr_assert /* Simulate User Assert */\n");
-	dbg("sysdbg dump_stack /* Dumps the current task stack */\n");
-	dbg("sysdbg dump_allstack /* Dumps the stack of all tasks */\n");
-	dbg("sysdbg help /* Display the help */\n");
+	dbg("\n");
+	dbg(" ______________________________________________________________________________\n");
+	dbg("| USAGE: sysdbg OPTION [COUNT]\n");
+	dbg("|  OPTION       -     Description \n");
+	dbg("|______________________________________________________________________________\n");
+	dbg("|     1         -  enable_monitor  : Allocates memory for debug struct & enables sysdbg monitoring\n");
```

Comment:
How about changing the option with defined value like `SYSDBG_ENABLE_MONITOR`?
For all options.

Reply:
This is possible but main intent here was to ease the user overhead of remembering the options in form of string.

Reply:
I mean, only changing `1` to `"%d - enable monitor ...", SYSDBG_ENABLE_MONITOR`.
Then from the log, it shows the number, so easy to understand.

Reply:
ok we will update that.

Reply:
Thanks. With defined value, it is better to maintain the code even if those values are changed.